src/screens/SettingsScreen.js

```diff
@@ -207,7 +212,7 @@ export const SettingsScreen = ({ navigation, route }) => {
         settingsList.push({
           data: [
             {
-              title: texts.settingsTitles.analytics,
+              title: settingsScreenListItemTitles.matomo || texts.settingsTitles.analytics,
```

Comment:
why a different naming here with `matomo` instead of `analytics`? would be nice to keep it consistent with the others

Reply:
i see, we already have inconsistencies there :(